docs/contribute/bors.md

```diff
@@ -60,10 +60,11 @@ more details.)
 
 - Commit messages are taken from the first post of the PR. This is much nicer than mergify /
   Github's default which just combines all commit subject lines into one. We have set things
-  up so that the text after "TO CONTRIBUTORS" is automatically ignored. Note that
-  contributions from other users will have to be added explicitly to the first post with
-  ["Co-authored-by:" trailers](https://github.blog/2018-01-29-commit-together-with-co-authors/);
-  they aren't automatically added like how mergify and Github previously did.
+  up so that the text after the first occurrence of "`---`" is automatically ignored. Contributions
+  from other users should be automatically added as [co-authors](https://github.blog/2018-01-29-commit-together-with-co-authors/),
+  like how mergify and Github previously did. For PRs with a lot of commits (100+) they
```

Comment:
This sounds suspiciously specific, did I miss a 100+ commit PR recently?

Reply:
I guess it's more like 50+ than 100+, actually. It's just something that I've noticed that I figured I'd record somewhere. Let me edit.

Reply:
I think the number is 250, there is no limit in bors but the github API call has a maximum: https://developer.github.com/v3/pulls/#list-commits-on-a-pull-request

Reply:
See for example #1564 (81 commits), #2714 (42 commits).